docs/developer-docs/latest/developer-resources/cli/CLI.md

```diff
@@ -6,7 +6,11 @@ canonicalUrl: https://docs.strapi.io/developer-docs/latest/developer-resources/c
 
 # Command Line Interface (CLI)
 
-Strapi comes with a full featured Command Line Interface (CLI) which lets you scaffold and manage your project in seconds.
+Strapi comes with a full featured Command Line Interface (CLI) which lets you scaffold and manage your project in seconds. 
+
+::: note
+We recommend to install strapi only locally and never globally. The following documentation does not use any required node prefixes like `npm run` or `yarn`, but you should always prefix `strapi` with what fits best to your project setup, e.g. `npm run strapi new <name>`, `yarn strapi new <name>`
```

Comment:
```suggestion
We recommend installing strapi only locally and never globally. The following documentation does not use any required node prefixes (i.e. `npm run` or `yarn`), but commands should prefix `strapi` with what fits best to the project setup (e.g. `npm run strapi new <name>`, `yarn strapi new <name>`).
```

Reply:
Suggesting this to be compliant with our [12 rules of technical writing](https://handbook.strapi.io/user-success-manual/12-rules-of-technical-writing)

Reply:
@pwizla Excellent! Thank you very much!